Content.Client/SimpleStation14/Examine/CharacterInformation/UI/CharacterInformationWindow.xaml

```diff
@@ -0,0 +1,35 @@
+<controls:FancyWindow xmlns="https://spacestation14.io"
+    xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
+    xmlns:graphics="clr-namespace:Robust.Client.Graphics;assembly=Robust.Client"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:style="clr-namespace:Content.Client.Stylesheets"
+    Title="{Loc 'character-information-ui-title'}" MinSize="675 384" Resizable="True">
+    <BoxContainer Orientation="Horizontal" HorizontalExpand="True" VerticalExpand="True" Margin="8">
+        <BoxContainer Orientation="Vertical">
+            <GridContainer Name="SpriteContainer" Columns="2">
+                <!-- Added by code
+                <SpriteView Name="SpriteViewS" />
+                <SpriteView Name="SpriteViewN" />
+                <SpriteView Name="SpriteViewW" />
+                <SpriteView Name="SpriteViewE" />
```

Comment:
I'm still of the opinion that all directions is overkill. 
Maybe we can do a poll on it at some point :P